Index: gr-soapy-1.0.2/CMakeLists.txt
===================================================================
--- gr-soapy-1.0.2.orig/CMakeLists.txt
+++ gr-soapy-1.0.2/CMakeLists.txt
@@ -49,7 +49,7 @@ list(INSERT CMAKE_MODULE_PATH 0 ${CMAKE_
 set(VERSION_INFO_MAJOR_VERSION 1)
 set(VERSION_INFO_API_COMPAT    0)
 set(VERSION_INFO_MINOR_VERSION 2)
-set(VERSION_INFO_MAINT_VERSION git)
+set(VERSION_INFO_MAINT_VERSION 0)

 # Set cmake policies.
 # This will suppress developer warnings during the cmake process that can occur
